{
  "Class": "patch",
  "ID": "oval:com.redhat.rhsa:def:20243543",
  "Version": "634",
  "Metadata": {
    "Title": "RHSA-2024:3543: python-idna security and bug fix update (Moderate)",
    "AffectedList": [
      {
        "Family": "unix",
        "Platforms": [
          "Red Hat Enterprise Linux 8.8 Extended Update Support"
        ]
      }
    ],
    "References": [
      {
        "Source": "RHSA",
        "RefID": "RHSA-2024:3543",
        "RefURL": "https://access.redhat.com/errata/RHSA-2024:3543"
      },
      {
        "Source": "CVE",
        "RefID": "CVE-2024-3651",
        "RefURL": "https://access.redhat.com/security/cve/CVE-2024-3651"
      }
    ],
    "Description": "Security Fix(es):\n\n* python-idna: potential DoS via resource consumption via specially crafted\ninputs to idna.encode() (CVE-2024-3651)",
    "Advisory": {
      "From": "secalert@redhat.com",
      "Severity": "Moderate",
      "Rights": "Copyright 2024 Red Hat, Inc.",
      "Issued": {
        "Date": "2024-06-03"
      },
      "Updated": {
        "Date": "2024-06-03"
      },
      "Cves": [
        {
          "CveID": "CVE-2024-3651",
          "Cvss3": "6.5/CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H",
          "Cwe": "CWE-400",
          "Impact": "moderate",
          "Href": "https://access.redhat.com/security/cve/CVE-2024-3651",
          "Public": "20240412"
        }
      ],
      "Bugzilla": [
        {
          "Href": "https://bugzilla.redhat.com/2274779",
          "ID": "2274779"
        }
      ],
      "AffectedCpeList": [
        "cpe:/a:redhat:rhel_eus:8.8",
        "cpe:/a:redhat:rhel_eus:8.8::appstream",
        "cpe:/a:redhat:rhel_eus:8.8::crb",
        "cpe:/a:redhat:rhel_eus:8.8::highavailability",
        "cpe:/a:redhat:rhel_eus:8.8::nfv",
        "cpe:/a:redhat:rhel_eus:8.8::realtime",
        "cpe:/a:redhat:rhel_eus:8.8::resilientstorage",
        "cpe:/a:redhat:rhel_eus:8.8::sap",
        "cpe:/a:redhat:rhel_eus:8.8::sap_hana",
        "cpe:/a:redhat:rhel_eus:8.8::supplementary",
        "cpe:/o:redhat:rhel_eus:8.8",
        "cpe:/o:redhat:rhel_eus:8.8::baseos"
      ],
      "Affected": {
        "Resolution": {}
      }
    }
  },
  "Criteria": {
    "Operator": "OR",
    "Criterions": [
      {
        "Comment": "Red Hat Enterprise Linux must be installed",
        "TestRef": "oval:com.redhat.rhba:tst:20191992005"
      }
    ],
    "Criterias": [
      {
        "Operator": "AND",
        "Criterions": [
          {
            "Comment": "python3-idna is earlier than 0:2.5-5.el8_8.1",
            "TestRef": "oval:com.redhat.rhsa:tst:20243543001"
          },
          {
            "Comment": "python3-idna is signed with Red Hat redhatrelease2 key",
            "TestRef": "oval:com.redhat.rhsa:tst:20243543002"
          }
        ],
        "Criterias": [
          {
            "Operator": "OR",
            "Criterions": [
              {
                "Comment": "Red Hat Enterprise Linux 8 is installed",
                "TestRef": "oval:com.redhat.rhba:tst:20191992003"
              },
              {
                "Comment": "Red Hat CoreOS 4 is installed",
                "TestRef": "oval:com.redhat.rhba:tst:20191992004"
              }
            ]
          }
        ]
      }
    ]
  }
}